src/pages/proposal-commons/proposal-list/proposal-status/proposal-status.tsx

```diff
@@ -47,11 +49,21 @@ const ProposalStatus = (props: Props) => {
           className={styles.execute}
           onClick={async () => {
             if (!voting) return;
-            const tx = await voting[proposal.type].executeVote(proposal.voteId);
+            const goTransaction = await go(voting[proposal.type].executeVote(proposal.voteId));
+            if (!isGoSuccess(goTransaction)) {
+              if (isUserRejection(goTransaction[GO_ERROR_INDEX])) {
+                notifications.info({ message: messages.TX_GENERIC_REJECTED });
+                return;
+              }
+              notifications.error({
+                message: messages.TX_GENERIC_ERROR,
+                errorOrMessage: goTransaction[GO_ERROR_INDEX],
+              });
+              return;
+            }
             setChainData('Save execute transaction', (state) => ({
-              transactions: [...state.transactions, { type: 'execute', tx }],
+              transactions: [...state.transactions, { type: 'execute', tx: goTransaction[GO_RESULT_INDEX] }],
             }));
-            await voting[proposal.type].executeVote(proposal.voteId);
```

Comment:
I guess up until now we double voted???

Reply:
It does look like the contract checks `vote._executed` so I think this second `executeVote` would fail every time?

Reply:
Yeah, probably